semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/HasScalaSigVisitor.scala

```diff
@@ -0,0 +1,15 @@
+package scala.meta.internal.metacp
+
+import scala.tools.asm.Attribute
+import scala.tools.asm.ClassVisitor
+import scala.tools.asm.Opcodes
+
+final class HasScalaSigVisitor extends ClassVisitor(Opcodes.ASM5) {
```

Comment:
I'd prefer to remove this class and use `ClassNode` to: 1) be consistent with how things are done in Main, 2) shrink the conceptual surface of the module.

Reply:
Removed.